src/main/java/com/uid2/admin/auth/AdminAuthMiddleware.java

```diff
@@ -16,32 +17,47 @@ public class AdminAuthMiddleware {
     private final AuthProvider authProvider;
     private final String environment;
     private final boolean isAuthDisabled;
+
+    final Map<Role, List<OktaGroup>> roleToOktaGroups = new EnumMap<>(Role.class);
     public AdminAuthMiddleware(AuthProvider authProvider, JsonObject config) {
         this.authProvider = authProvider;
         this.environment = config.getString("environment", "local");
         this.isAuthDisabled = config.getBoolean("is_auth_disabled", false);
+        roleToOktaGroups.put(Role.MAINTAINER, parseOktaGroups(config.getString(AdminConst.ROLE_OKTA_GROUP_MAP_MAINTAINER)));
+        roleToOktaGroups.put(Role.PRIVILEGED, parseOktaGroups(config.getString(AdminConst.ROLE_OKTA_GROUP_MAP_PRIVILEGED)));
+        roleToOktaGroups.put(Role.SUPER_USER, parseOktaGroups(config.getString(AdminConst.ROLE_OKTA_GROUP_MAP_SUPER_USER)));
+    }
+
+    private List<OktaGroup> parseOktaGroups(final String oktaGroups) {
+        final List<OktaGroup> allOktaGroups = new ArrayList<>();
+        for (String group : oktaGroups.split(",")) {
```

Comment:
Should we remove whitespace when splitting in case someone does something like:
- name: role_okta_group_map_super_user
   value: "developer-elevated, admin"

Reply:
good call. I'll update this